docs/local-authority-changes.md

```diff
@@ -0,0 +1,22 @@
+# Local Authority Changes
+
+Local Links Manager is set up to edit links, and editing other aspects of Local Authorities is not really supported by the front end. Fortunately these changes do not happen often, but when they do they tend to happen in batches.
+
+## To change a Local Authority's details
+
+The URL can be changed in the UI, but if you need to change the slug and/or authority name, you'll have to log on to the app-console to do it.
+
+## To mark a Local Authority as superseded
+
+Sometimes local authorities are merged (this tends to be borough/city/district councils getting rolled up into their county council, which then becomes a unitary authority). In this case, you should create a rake task that does the following:
+
+- Update the county council from a tier 2 (county) to a tier 3 (unitary) council (or create a new unitary council if repurposing the previous one is likely to cause problems).
+- For each of the councils being merged:
+- - Set the active_end_date property to the date on which it will be (or aws, if you're doing this in retrospect) merged into the unitary authority.
```

Comment:
In the rendered version the nested list items seem to display with double bullets. I think you can format them like so:
```
- For each of the councils being merged:
  - Set the active_end_date property to the date on which it will be (or aws, if you're doing this in retrospect) merged into the unitary authority.
```